File: app/src/main/java/me/dhivo/android/pingpongmatchtracker/fragments/SettingsFragment.java
package me.dhivo.android.pingpongmatchtracker.fragments;

import android.os.Bundle;
import android.preference.PreferenceFragment;

import me.dhivo.android.pingpongmatchtracker.R;

public class SettingsFragment extends PreferenceFragment {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.settings);
    }
}

File: app/src/main/java/me/dhivo/android/pingpongmatchtracker/fragments/SettingsFragment.kt
package me.dhivo.android.pingpongmatchtracker.fragments

import android.os.Bundle
import android.preference.PreferenceFragment

import me.dhivo.android.pingpongmatchtracker.R

class SettingsFragment : PreferenceFragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.settings)
    }
}